packages/components/psammead-social-embed/src/Notice/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { memo } from 'react';
```

Comment:
Please could you add a small section to the readme stating that we use React.memo and then a link to the react docs.